src/smoltcp_phy.rs

```diff
@@ -80,7 +73,7 @@ pub struct EthRxToken<'a> {
     len: usize
 }
 
-impl<'a> RxToken for EthRxToken<'a> {
```

Comment:
This space belongs here.

Reply:
Thanks! Fixed with 669fe7ab9be7591dbd4f0fba243f14a00acf8fae.

Reply:
Forced-pushed branch to fix some messy code and add delays (see #3).